server/app.js

```diff
@@ -2,19 +2,27 @@ const express = require('express');
 const helmet = require('helmet');
 const morgan = require('morgan');
 const path = require('path');
+const passport = require('passport');
+const bodyParser = require('body-parser');
+
 
 const errorHandler = require(path.resolve(__dirname, './utils/errorHandling'));
 const { stream } = require('./utils/logger');
 const connectDB = require('./database');
-// const apiV1Router = require('./routes/v1.routes');
+const userRouter = require('./routes/v1.routes/userRouter');
```

Comment:
Since it's best practice to use api versioning we should define those routes in our v1.routes.js file which is the only we need in app.js. We need to define all the routes, including auth routes inside our v1.routes.js file. We should go even a step further and create a route file for auth which is going to be redirected from v1.routes.js and should have only the routes for auth.